File: Python/2941.py
import sys

f = lambda : sys.stdin.readline().rstrip()

list = ["c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="]

line = f()

for i in list:

    if i in line:

        line = line.replace(i, "#")

print(len(line))

File: Java/_2941.java
package Algorithm.BaekJoon;

import java.io.*;
import java.util.*;

public class _2941 {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        List<String> list = Arrays.asList("c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z=");

        String str = br.readLine().trim();

        for (String key : list) {

            if (str.contains(key)) {

                str = str.replace(key, "#");
            }
        }

        System.out.println(str.length());
    }
}